m2c/types.py

```diff
@@ -942,6 +952,12 @@ def reg64(*, likely_float: bool) -> Type:
         likely = TypeData.K_FLOAT if likely_float else TypeData.K_INT
         return Type(TypeData(kind=kind, likely_kind=likely, size_bits=64))
 
+    @staticmethod
+    def reg128(*, likely_float: bool) -> Type:
```

Comment:
I'd remove the likely_float argument and use kind = K_INT, unless there are 128-bit floats